### ISSUE TITLE
Form group on the “My Health Log Symptoms List” screen is not programmatically associated with its group label [iOS]

### ISSUE THREAD

**Issue by willslone** [covid-alert-ny/covid-green-app#262]
The “Do you have any of the following 11 symptoms today?” form group label is not announced upon focusing a control in the form group. For example, the “Fatigue” checkbox is announced as “Fatigue, unselected, checkbox”.

![](https://user-images.githubusercontent.com/4437084/92046290-39234200-ed50-11ea-8759-34c094e65f96.png)

Use the accessibilityLabel property to include the group label in the accessible name of each control. Remove the group label from the reading order using the accessibilityElementsHidden, as it is redundant when included in the accessibilityLabel of each checkbox.

```javascript
<View accessibilityElementsHidden={true}>
  <Text>Do you have any of the following 11 symptoms today?</Text>
</View>

<TouchableOpacity
  onPress={…}
  accessibilityLabel='Fatigue, Do you have any of the following 11 symptoms today?'
  accessibilityRole='checkbox'
  accessibilityState={{selected: false}}
>
```

**Comment by floridemai** [comment]
@willslone This has now been addressed
